src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -224,13 +237,18 @@ private void FileWorkItemsHelper(SarifLog sarifLog, SarifWorkItemContext filingC
                 //
                 UpdateLogWithWorkItemDetails(sarifLog, sarifWorkItemModel.HtmlUri, sarifWorkItemModel.Uri);
 
-                LogMetricsForFiledWorkItem(sarifLog, sarifWorkItemModel, FilingResult.Succeeded);
+                LogMetricsForProcessedModel(sarifLog, sarifWorkItemModel, FilingResult.Succeeded, null);
             }
             catch (Exception ex)
             {
                 this.Logger.LogError(ex, "An exception was raised filing log '{logGuid}'.", logGuid);
                 Console.Error.WriteLine(ex);
-                LogMetricsForFiledWorkItem(sarifLog, sarifWorkItemModel, FilingResult.ExceptionRaised);
+
+                Dictionary<string, object> customDimentions = new Dictionary<string, object>();
```

Comment:
>string [](start = 27, length = 6)

You should put this whole catch block in a helper that accepts the exception and the model.